x/superfluid/types/errors.go

```diff
@@ -51,3 +52,54 @@ type LockOwnerMismatchError struct {
 func (e LockOwnerMismatchError) Error() string {
 	return fmt.Sprintf("lock ID %d owner %s does not match provided owner %s.", e.LockId, e.LockOwner, e.ProvidedOwner)
 }
+
```

Comment:
I think some of the new errors introduced here are being unused, would you be able to double check if this is the case and if so erase unused errors?

Reply:
Removed unused errors here https://github.com/osmosis-labs/osmosis/pull/5178/commits/a9d0c319463b5161d4114b4cab66f485ddf0c249

Reply:
(they are used in part 2 of this PR so I just added them back there, thanks for the catch though)